### PR TITLE
Added section number to "see also" references

### PR DESCRIPTION
**Pull request by polluks**

### CODE REVIEW

File: pipetty.c [hunk 1]
```diff
@@ -103,7 +103,7 @@ int main(int argc, const char **argv)
             dup2(p[0], 0);
             close(p[0]);
 
-            execlp("less", "less", "-R", "-", 0);
+            execlp("less", "less", "-R", "-", NULL);
```
Comment (kilobyte) [inline pipetty.c]: A compiler insisting on writing `0` as `NULL`?  Without even a semantic change, like `nullptr`... That's weird.

**Comment by kilobyte** [comment]
Merged, thanks!

File: pipetty.c (hunk 1 above)
Comment (polluks) [reply pipetty.c]: At least gcc 7.5 recommends it. Thanks for merging :-)

Comment (kilobyte) [reply pipetty.c]: Ah, `-Wall` got lost when I converted to CMake, thanks for pointing out the warning.

**Pull request closed without merging**